### PR TITLE
Feature / support new size in AndesProgressIndicatorIndeterminate

### PR DESCRIPTION
**Pull request by javff**
## Description
Add `xSmall`  to `ProgressIndicatorSize`

Describe why we need these changes in the lib.
Try to answer these questions:
- What is it about?
- How does it work?
## Affected component
This PR affect `AndesProgressIndicatorIndeterminate`
## RFC Link
https://docs.google.com/document/d/1C8anm9CazQRBiVPrvKoSEvwzYaldiZ-RosdPDtOLIoE/edit#
## Checks
Are you sure you followed all those steps? If so, repeat after me "I solemnly swear that ...":
   - [X] I have coded an example inside the Demo App,
   - [X] I have added proper tests,
   - [x] I have modified the Changelog.md file,
   - [ ] I have updated GitHub wiki,
   - [x] I have the explicit approval of one or more members of the UX Team,
   - [x] I have checked that this code is ObjC compliant.
   - [X] I have checked that all the new public enums conform to AndesEnumStringConvertible.
## Change type
This is easy, let us know what this code is about:
   - [ ] This code adds a new component
   - [X] This code includes improvements to an existent component
   - [ ] This code improves or modifies ONLY the Demo App.

### CODE REVIEW

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift [hunk 1]
```diff
@@ -14,12 +14,16 @@ import Foundation
...
+    case xSmall
+    case xxSmall
 
     public static func keyFor(_ value: AndesProgressIndicatorSize) -> String {
         switch value {
         case .large: return "LARGE"
         case .medium: return "MEDIUM"
         case .small: return "SMALL"
+        case .xSmall: return "XSMALL"
+        case .xxSmall: return "XXSMALL"
```
Comment (tinchovictory) [inline LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift:26]: Borremos este
```suggestion
```

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift [hunk 2]
```diff
@@ -14,12 +14,16 @@ import Foundation
     case large
     case medium
     case small
+    case xSmall
+    case xxSmall
```
Comment (tinchovictory) [inline LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift:18]: Borremos este
```suggestion
```

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeFactory.swift [hunk 3]
```diff
@@ -19,6 +19,10 @@ class AndesProgressIndicatorSizeFactory {
             return AndesProgressIndicatorSizeMedium()
         case .small:
             return AndesProgressIndicatorSizeSmall()
+        case .xSmall:
+            return AndesProgressIndicatorSizeXSmall()
+        case .xxSmall:
+            return AndesProgressIndicatorSizeXXSmall()
```
Comment (tinchovictory) [inline LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeFactory.swift:25]: Borremos este
```suggestion
```

File: Example/Tests/AndesProgressIndicatorIndeterminateTest.swift [hunk 4]
```diff
@@ -74,6 +76,39 @@ class AndesProgressIndicatorIndeterminateTest: QuickSpec {
...
+                    }
+
+                    let expectedFont = AndesStyleSheetManager.styleSheet.regularSystemFont(size: 14)
+
+                    expect(contentView.config.size?.height).to(equal(12))
+                    expect(contentView.config.size?.strokeWidth).to(equal(2))
+                    expect(contentView.config.size?.textSpacing).to(equal(12))
+                    expect(contentView.config.size?.horizontalDistribution).to(beTrue())
                     expect(contentView.config.size?.textFont).to(equal(expectedFont))
                 }
```
Comment (tinchovictory) [inline Example/Tests/AndesProgressIndicatorIndeterminateTest.swift:113]: Borremos este caso entero
```suggestion
```

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeXXSmall.swift [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+internal struct AndesProgressIndicatorSizeXXSmall: AndesProgressIndicatorSizeProtocol {
+    var textSpacing: CGFloat = 12
+    var strokeWidth: CGFloat = 2
+    var height: CGFloat = 12
+    var horizontalDistribution: Bool = true
+
+    var textFont: UIFont = AndesStyleSheetManager.styleSheet.regularSystemFont(size: 14)
+
+}
```
Comment (tinchovictory) [inline LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeXXSmall.swift:18]: Borremos este archivo entero

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift [hunk 6]
```diff
@@ -14,10 +14,14 @@ class ProgressIndicatorViewController: UIViewController {
     @IBOutlet weak var progressIndicatorLargeWithText: AndesProgressIndicatorIndeterminate!
     @IBOutlet weak var progressIndicatorSmallWithText: AndesProgressIndicatorIndeterminate!
     @IBOutlet weak var progressIndicatorMediumWithText: AndesProgressIndicatorIndeterminate!
+    @IBOutlet weak var progressIndicatorXSmallWithText: AndesProgressIndicatorIndeterminate!
+    @IBOutlet weak var progressIndicatorXXSmallWithText: AndesProgressIndicatorIndeterminate!
 
     @IBOutlet weak var progressIndicatorLargeWithoutText: AndesProgressIndicatorIndeterminate!
     @IBOutlet weak var progressIndicatorSmallWithoutText: AndesProgressIndicatorIndeterminate!
     @IBOutlet weak var progressIndicatorMediumWithoutText: AndesProgressIndicatorIndeterminate!
+    @IBOutlet weak var progressIndicatorXXSmallWithoutText: AndesProgressIndicatorIndeterminate!
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift:23]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift [hunk 7]
```diff
@@ -14,10 +14,14 @@ class ProgressIndicatorViewController: UIViewController {
     @IBOutlet weak var progressIndicatorLargeWithText: AndesProgressIndicatorIndeterminate!
     @IBOutlet weak var progressIndicatorSmallWithText: AndesProgressIndicatorIndeterminate!
     @IBOutlet weak var progressIndicatorMediumWithText: AndesProgressIndicatorIndeterminate!
+    @IBOutlet weak var progressIndicatorXSmallWithText: AndesProgressIndicatorIndeterminate!
+    @IBOutlet weak var progressIndicatorXXSmallWithText: AndesProgressIndicatorIndeterminate!
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift:18]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift [hunk 8]
```diff
@@ -38,20 +42,28 @@ class ProgressIndicatorViewController: UIViewController {
         progressIndicatorLargeWithText.startAnimation()
         progressIndicatorSmallWithText.startAnimation()
         progressIndicatorMediumWithText.startAnimation()
+        progressIndicatorXSmallWithText.startAnimation()
+        progressIndicatorXXSmallWithText.startAnimation()
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift:46]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift [hunk 9]
```diff
@@ -38,20 +42,28 @@ class ProgressIndicatorViewController: UIViewController {
...
 
         progressIndicatorLargeWithoutText.startAnimation()
         progressIndicatorSmallWithoutText.startAnimation()
         progressIndicatorMediumWithoutText.startAnimation()
+        progressIndicatorXSmallWithoutText.startAnimation()
+        progressIndicatorXXSmallWithoutText.startAnimation()
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift:52]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift [hunk 10]
```diff
@@ -38,20 +42,28 @@ class ProgressIndicatorViewController: UIViewController {
...
     }
 
     private func stopAnimation() {
         progressIndicatorLargeWithText.stopAnimation()
         progressIndicatorSmallWithText.stopAnimation()
         progressIndicatorMediumWithText.stopAnimation()
+        progressIndicatorXSmallWithText.stopAnimation()
+        progressIndicatorXXSmallWithText.stopAnimation()
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift:60]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift [hunk 11]
```diff
@@ -38,20 +42,28 @@ class ProgressIndicatorViewController: UIViewController {
...
 
         progressIndicatorLargeWithoutText.stopAnimation()
         progressIndicatorSmallWithoutText.stopAnimation()
         progressIndicatorMediumWithoutText.stopAnimation()
+        progressIndicatorXSmallWithoutText.stopAnimation()
+        progressIndicatorXXSmallWithoutText.stopAnimation()
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorViewController.swift:66]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m [hunk 12]
```diff
@@ -14,10 +14,17 @@ @interface ProgressIndicatorObjcViewController ()
 @property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorMediumWithText;
 @property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorSmallWithText;
 
+@property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorXSmallWithText;
+@property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorXXSmallWithText;
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m:18]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m [hunk 13]
```diff
@@ -14,10 +14,17 @@ @interface ProgressIndicatorObjcViewController ()
...
+
+
 @property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorLargeWithoutText;
 @property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorMediumWithoutText;
 @property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorSmallWithoutText;
 
+@property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorXSmallWithoutText;
+@property (weak, nonatomic) IBOutlet AndesProgressIndicatorIndeterminate *progressIndicatorXXSmallWithoutText;
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m:26]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m [hunk 14]
```diff
@@ -42,20 +49,30 @@ - (void) startAnimation {
     [self.progressIndicatorLargeWithText startAnimation];
     [self.progressIndicatorMediumWithText startAnimation];
     [self.progressIndicatorSmallWithText startAnimation];
+    [self.progressIndicatorXSmallWithText startAnimation];
+    [self.progressIndicatorXXSmallWithText startAnimation];
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m:53]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m [hunk 15]
```diff
@@ -42,20 +49,30 @@ - (void) startAnimation {
...
+
     
     [self.progressIndicatorLargeWithoutText startAnimation];
     [self.progressIndicatorMediumWithoutText startAnimation];
     [self.progressIndicatorSmallWithoutText startAnimation];
+    [self.progressIndicatorXSmallWithoutText startAnimation];
+    [self.progressIndicatorXXSmallWithoutText startAnimation];
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m:60]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m [hunk 16]
```diff
@@ -42,20 +49,30 @@ - (void) startAnimation {
...
+
 }
 
 - (void)stopAnimation {
     [self.progressIndicatorLargeWithText stopAnimation];
     [self.progressIndicatorMediumWithText stopAnimation];
     [self.progressIndicatorSmallWithText stopAnimation];
+    [self.progressIndicatorXSmallWithText stopAnimation];
+    [self.progressIndicatorXXSmallWithText stopAnimation];
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m:69]:
```suggestion
```

File: Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m [hunk 17]
```diff
@@ -42,20 +49,30 @@ - (void) startAnimation {
...
     
     [self.progressIndicatorLargeWithoutText stopAnimation];
     [self.progressIndicatorMediumWithoutText stopAnimation];
     [self.progressIndicatorSmallWithoutText stopAnimation];
+    [self.progressIndicatorXSmallWithoutText stopAnimation];
+    [self.progressIndicatorXXSmallWithoutText stopAnimation];
```
Comment (tinchovictory) [inline Example/AndesUI/ProgressIndicator/Views/ProgressIndicatorObjcViewController.m:75]:
```suggestion
```

**Review by tinchovictory — COMMENTED** [review]
Perdon por los cambios 🙈. Creo que con borrar esto estariamos. Solo quedaria cambiar los xibs para que dejen de tener la vista de xxsmall

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift (hunk 1 above)
Comment (javff) [reply LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift]: done !!

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift (hunk 2 above)
Comment (javff) [reply LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSize.swift]: done !!

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeFactory.swift (hunk 3 above)
Comment (javff) [reply LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeFactory.swift]: done !!

File: Example/Tests/AndesProgressIndicatorIndeterminateTest.swift (hunk 4 above)
Comment (javff) [reply Example/Tests/AndesProgressIndicatorIndeterminateTest.swift]: done !!

File: LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeXXSmall.swift (hunk 5 above)
Comment (javff) [reply LibraryComponents/Classes/Core/AndesProgressIndicator/Size/AndesProgressIndicatorSizeXXSmall.swift]: done !!

**Review by tinchovictory — APPROVED** [review]

**Pull request merged**